tests/apitest/src/CoreGraphics/CGDisplay.cs

```diff
@@ -0,0 +1,99 @@
+﻿using NUnit.Framework;
+using System;
+
+#if !XAMCORE_2_0
+using MonoMac.AppKit;
+using MonoMac.ObjCRuntime;
+using MonoMac.CoreImage;
+using MonoMac.CoreGraphics;
+using CGSize = System.Drawing.SizeF;
+using CGRect = System.Drawing.RectangleF;
+#else
+using AppKit;
+using ObjCRuntime;
+using CoreImage;
+using CoreGraphics;
+#endif
+
+namespace Xamarin.Mac.Tests
+{
+	[TestFixture]
+	public class CGDisplayTests
+	{
+		int mainDisplayId;
+
+		[SetUp]
+		public void Setup ()
+		{
+			mainDisplayId = CGDisplay.MainDisplayID;
+		}
+
+		[Test]
+		public void CGDisplayShouldGetDimensions ()
+		{
+			var bounds = CGDisplay.GetBounds (mainDisplayId);
+			Assert.IsFalse (bounds.IsNull ());
+			Assert.IsFalse (bounds.IsEmpty);
+
+			var width = CGDisplay.GetWidth (mainDisplayId);
+			Assert.Greater (0, width);
+
+			var height = CGDisplay.GetHeight (mainDisplayId);
+			Assert.Greater (0, height);
+		}
+
+		[Test]
+		public void CGDisplayCursorMethods ()
+		{
+			int error;
+			error = CGDisplay.HideCursor (mainDisplayId);
```

Comment:
Maybe this shouldn't be part of apitest? I don't want my mouse curser to flicker just because I ran mac-test while working on something else.